nil/cmd/sync_committee_cli/main.go

```diff
@@ -136,6 +137,41 @@ func buildDecodeBatchCmd(_ *commands.ExecutorParams, logger logging.Logger) *cob
 	return cmd
 }
 
+func buildResetContractCmd(_ *commands.ExecutorParams, logger logging.Logger) *cobra.Command {
+	params := &commands.ResetStateParams{}
+
+	cmd := &cobra.Command{
+		Use:   "reset-state",
+		Short: "Reset L1 state to specified root",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return commands.ResetState(context.Background(), params, logger)
+		},
+	}
+
+	cmd.Flags().StringVar(
```

Comment:
The default values are empty, so let's make these flags required (`MarkFlagRequired`)